index.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title> CLARENCE PORTFOLIO</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
+</head>
+
+
+<body>
+  
+  <header class="header" id="header">
+    <a href="#">CLARENCE</a>
+    <nav>
+      <span class="open" id="menu"></span>
+       <img src="images/Icon - Menu.png" alt="menu bar" class="icon-container">
+      
+    </nav>
+  </header>
+  
+  <!---Main---->
+
+  <main class="main">
+      
+    
+          <div class="home-page">
+              <div>
+                  <h2 class="home_header">Hey There.</h2>
+                  <h3 class="home_header">I'm Clarence Atim</h3>
```

Comment:
I think it's more semantic to use a `<h1>` tag for this title. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

 You may not really need to use different tags but you could use a `<br>` tag to break them into two lines.

Reply:
> I think it's more semantic to use a `<h1>` tag for this title. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
> 
> You may not really need to use different tags but you could use a `<br>` tag to break them into two lines.

- [ ] As sufficiently explained by the previous reviewer, he recommends you do something like:
```

  <h1 class="home_header">Hey There. <br>
    I'm Clarence Atim</h1>

```

with this, the title would be google accessible and even other devices can understand.